tests/integration/endpointRoutes.js

```diff
@@ -0,0 +1,352 @@
+'use strict'
+
+const request = require('supertest')
+const tap = require('tap')
+
+const {withTestMapperServer} = require('../utils')
+
+const port = 13008
+process.env.MONGO_URL = 'mongodb://localhost:27017/test'
```

Comment:
@bradsawadye This test suite is not cleaning up after itself. You've masked this by using the default test DB and you're relying on the unit tests to clean up the data created here. 

> Try running the test alone or as part of the `yarn test:integration`

The reason for this is that you are importing your utils before setting the DB Env Var. The method in utils takes the Env Vars from that moment in time. Ie: not the Mongo Url you thought. Therefore, you are creating two different DB connections.

```suggestion
const port = 13008
process.env.MONGO_URL = 'mongodb://localhost:27017/endpointRoutesTest'

const {withTestMapperServer} = require('../utils')
```